FlowCrypt/Functionality/Services/EnterpriseServerApi.swift

```diff
@@ -32,7 +32,11 @@ extension EnterpriseServerApiError: LocalizedError {
 class EnterpriseServerApi: EnterpriseServerApiType {
 
     private enum Constants {
-        static let getActiveFesTimeout: TimeInterval = 4
+        /// 404 - Not Found
+        static let getToleratedHTTPStatuses = [404]
+        /// -1001 - request timed out, -1003 - сannot resolve host
+        static let getToleratedNSErrorCodes = [-1001, -1003]
+        static let getActiveFesTimeout: TimeInterval = 100
```

Comment:
Also, what is the number 100 here, 100 seconds? Milliseconds?

Reply:
for some reason my lates commit wasn't included :/